v1/src/main/java/com/google/cloud/teleport/bigtable/CassandraToBigtable.java

```diff
@@ -213,7 +238,12 @@ public static void main(String[] args) {
             .withTable(options.getCassandraTable())
             .withMapperFactoryFn(cassandraObjectMapperFactory)
             .withEntity(Row.class)
-            .withCoder(SerializableCoder.of(Row.class));
+            .withCoder(SerializableCoder.of(Row.class))
+            .withQuery(
+                new CassandraWritetimeQueryProvider(
```

Comment:
CassandraWritetimeQueryProvider is not getting unpacked in directRunner.

Reply:
There's a bug in Beam cassandraIO: https://github.com/apache/beam/issues/31622

I'm working on a fix.

Reply:
https://github.com/apache/beam/pull/31623 fix waiting for review and Beam release. Integration testing can continue after this beam release.